doc/cli/npm-install.md

```diff
@@ -3,15 +3,23 @@ npm-install(1) -- Install a package
 
 ## SYNOPSIS
 
-    npm install (with no args in a package dir)
-    npm install <tarball file>
-    npm install <tarball url>
-    npm install <folder>
-    npm install [@<scope>/]<name> [--save|--save-dev|--save-optional] [--save-exact]
+    npm install (with no args, in package dir)
+    npm install [@<scope>/]<name>
     npm install [@<scope>/]<name>@<tag>
     npm install [@<scope>/]<name>@<version>
     npm install [@<scope>/]<name>@<version range>
-    npm i (with any of the previous argument usage)
+    npm install <tarball file>
+    npm install <tarball url>
+    npm install <folder>
+
+    alias: npm i
+    common options: [--save|--save-dev|--save-optional] [--save-exact]
+
+    Can specify one or more: npm install ./foo.tgz bar@stable /some/folder
+
+    If no argument is supplied and ./npm-shrinkwrap.json is present,
```

Comment:
OK, I can pull it.  That means pulling it from the `usage` too, though -- is that desirable?